Mathlib/Order/Basic.lean

```diff
@@ -220,6 +220,14 @@ alias lt_of_eq_of_lt ← Eq.trans_lt
 alias lt_of_eq_of_lt' ← Eq.trans_gt
 #align eq.trans_gt Eq.trans_gt
 
+@[simp] lemma le_of_subsingleton [Subsingleton α] (a b : α) : a ≤ b ↔ True := by
+  cases Subsingleton.elim a b
+  simp
+
+@[simp] lemma not_lt_of_subsingleton [Subsingleton α] (a b : α) : a < b ↔ False := by
+  cases Subsingleton.elim a b
+  simp
```

Comment:
Is there any reason to keep the `↔`s? I'm pretty sure `simp_rw` and `simp` already know how to handle them, unless there's something special about inequalities; and they just add noise to the statement.
```suggestion
@[simp] lemma le_of_subsingleton [Subsingleton α] (a b : α) : a ≤ b := by
  cases Subsingleton.elim a b
  simp

@[simp] lemma not_lt_of_subsingleton [Subsingleton α] (a b : α) : ¬a < b := by
  cases Subsingleton.elim a b
  simp
```